webapp/components/notifications/Notification/Notification.vue

```diff
@@ -42,11 +51,26 @@ export default {
   },
   computed: {
     excerpt() {
-      return this.$filters.removeLinks(this.post.contentExcerpt)
+      const excerpt = this.post.id ? this.post.contentExcerpt : this.comment.contentExcerpt
+      return (
+        (!this.post.id ? '<b>Comment: </b>' : '') + excerpt.replace(/<(?:.|\n)*?>/gm, '').trim()
+      )
     },
     post() {
       return this.notification.post || {}
     },
+    comment() {
+      return this.notification.comment || {}
+    },
+    postParams() {
+      return {
+        id: this.post.id || this.comment.post.id,
+        slug: this.post.slug || this.comment.post.slug,
+      }
+    },
+    hashParam() {
+      return this.post.id ? {} : { hash: `#commentId-${this.comment.id}` }
```

Comment:
I was thinking about this lately: How about a generic behaviour for fragment identifiers? Can't we simply scroll to *any* html element which has a `scroll-id="blabla"`? We could use it for comments but maybe for other use-cases as well?

Reply:
You maybe mix up queries with hashtags here? @roschaefer 

A hashtag is always about scrolling to an anchor. Isn’t it?
The naming `commentId` is for the user readability. That they can get a cloud about what the hashtag is, even in case the scrolling does not work for any reason.